### PR TITLE
Add tokens minted to subgraph

### PR DESCRIPTION
**Pull request by alicjakujawa**
This PR adds in queries for tokensMinted action

**Added/Changes**
* Handler for TokensMinted event
* Correctly expose token on Colony entity
* Updated events with additional first argument (address)
* Added TokensMinted event and handler to subgraph.yaml

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -3,7 +3,7 @@ type Colony @entity {
   ensName: String
   orbitAddress: String
   colonyChainId: BigInt!
-  token: String # Address
+  token: Token # Address
```
Comment (rdig) [inline schema.graphql:6]: This is a nitpick, but since this is an object now, the `# Address` comment is wrong :)

**Review by rdig — APPROVED** [review]
All good here

File: src/mappings/colonyNetwork.ts [hunk 2]
```diff
@@ -24,8 +26,28 @@ export function handleColonyAdded(event: ColonyAdded): void {
     colony.metadataHistory = []
   }
 
+  let tokenAddress = event.params.token.toHexString()
+  let token = Token.load(tokenAddress)
+  if (token == null){
```
Comment (area) [inline src/mappings/colonyNetwork.ts:31]: We copied this block from [colony.ts](https://github.com/JoinColony/subgraph/blob/master/src/mappings/colony.ts#L56) to get things going - can we refactor so it only appears once?

File: schema.graphql (hunk 1 above)
Comment (area) [reply schema.graphql:6]: I've gone back and forth on this. You still set the property to the address, as the ID of the token is the token address, and the type says "there's an object of this type with a corresponding ID". I can go either way :smile:

File: src/mappings/colonyNetwork.ts (hunk 2 above)
Comment (alicjakujawa) [reply src/mappings/colonyNetwork.ts]: Good point, forgot about it, updated!

**Review by area — APPROVED** [review]

**Pull request merged**